spec/factories/organizations.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :organization do
+    name { Faker::Company.name }
+  end
+end
```

Comment:
Please check the EOL in all file, And make proper changes. Thank you!

Reply:
done bhaiya